cpp/src/copying/scatter.cu

```diff
@@ -97,7 +97,7 @@ void operator()(table const *source_table, gdf_column const& scatter_map,
 
   constexpr int block_size = 256;
   const gdf_size_type invert_grid_size =
-    (destination_table->num_rows() + block_size - 1) / block_size;
+    (source_table->num_rows() + block_size - 1) / block_size;
```

Comment:
This should use `cudf::util::round_up_safe()` to avoid overflow (since num_rows() can reach the limit of its data type).

Reply:
I get `Error: Internal Compiler Error (codegen): "there was an error in verifying the lgenfe output!"`

Reply:
Use https://github.com/rapidsai/cudf/blob/branch-0.10/cpp/src/utilities/cuda_utils.hpp#L40

Reply:
Done

Reply:
Yes, after I left the computer I thought "oops, that wasn't what I should have recommended!"